docs/values.md

```diff
@@ -89,6 +89,18 @@ The `blockedNamespaceList` lists the Kubernetes/Openshift namespaces blocked by
       - kube-system
       - kube-public
 
+### AKOSetttings.istioEnabled (Tech Preview)
+
+AKO can be deployed in Istio environment. Setting this to `true` indicates to AKO that the environment is Istio. Default value is `false`.
+
+### AKOSetttings.ipFamily (Tech Preview)
+
+`V6` is currently supported only for `vCenter` cloud with `calico` CNI.
+
+AKO can be deployed with ipFamily as `V4` or `V6`. When ipFamily is set to `V6`, AKO looks for `V6` IP for nodes from calico annotation and creates routes on controller. Only servers with `V6` IP will get added to Pools.
```

Comment:
what will be the value if the user wants to use dual-stack?

Reply:
dual stack for backend is not supported